Generator/Sources/NeedleFramework/Parsing/Tasks/ASTUtils.swift

```diff
@@ -40,7 +40,7 @@ extension Dictionary where Key: ExpressibleByStringLiteral {
 
     /// Check if this structure represents a `Dependency` protocol.
     var isDependencyProtocol: Bool {
-        return inheritedTypes.contains("Dependency")
+        return inheritedTypes.contains("Dependency") || inheritedTypes.contains("NeedleFoundation.Dependency")
```

Comment:
Might be time to create a global module name.  Can't see an obvious __MODULE_NAME__ type thing we could use.

Reply:
That's probably a good idea. Can we do that here in this PR?